l10n_br_fiscal/models/res_company.py

```diff
@@ -557,3 +516,50 @@ def _onchange_tax_inss_wh_id(self):
             self._set_tax_definition(self.tax_inss_wh_id)
         else:
             self._del_tax_definition(TAX_DOMAIN_INSS_WH)
+
+    @api.depends("annual_revenue", "payroll_amount")
+    def _compute_simplifed_tax(self):
+        for record in self:
+            record._calculate_coefficient_r()
+
+    def _compute_icmssn(self):
```

Comment:
Acho que seria informativo ter um docstring aqui para explicar, porque foi feito esse filtro

Reply:
No simples nacional a alíquota do ICMS quando é Comércio é um valor, quando é Industria é outro valor.
Eu uso o filtro para procurar o valor efetivo para quando for comércio e depois faço o mesmo para quando for indústria.
Esse compute está sendo usado para calcular essa duas variáveis : icmssn_credit_commerce, icmssn_credit_industry.
Eles são usados depois na emissão da nota fiscal do simples nacional.

Reply:
Eu vou pedir pro @renatonlima comentar de novo sobre esse issue...